kotlin-native/backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/BinaryOptions.kt

```diff
@@ -23,6 +23,8 @@ object BinaryOptions : BinaryOptionRegistry() {
     val sourceInfoType by option<SourceInfoType>()
 
     val androidProgramType by option<AndroidProgramType>()
+
+    val unitSuspendFunctionExport by option<UnitSuspendFunctionExport>()
```

Comment:
The option name should indicate that it affects only Obj-C export. E.g. `unitSuspendFunctionObjCExport`.